Marsiple/Comment/CommentViewController.swift

```diff
@@ -31,8 +31,8 @@ private extension CommentViewController {
 
 private extension CommentViewController {
     func setupNavigationBar() {
-        navigationItem.title = "Comment"  // TODO: Localize
-        let saveButton = UIBarButtonItem(title: "Save", style: .done, target: self, action: #selector(saveButtonTapped))
+        navigationItem.title = LocalizationKey.Comment.navigationBarTitle.localized()
+        let saveButton = UIBarButtonItem.createSaveItem(target: self, action: #selector(saveButtonTapped))
         navigationItem.rightBarButtonItem = saveButton
```

Comment:
The idea behind the `UIBarButtonItem` extension is to have as little code as possible.

 With that in mind, I would discard the 

```let saveButton = UIBarButtonItem.createSaveItem(target: self, action: #selector(saveButtonTapped))```

line and go for

```navigationItem.rightBarButtonItem = UIBarButtonItem.createSaveItem(target: self, action: #selector(saveButtonTapped))``` directly.